### PR TITLE
feat(CB2-12602): Display re-application date from test-results on IVA30 and MSVA30

### PR DESCRIPTION
**Pull request by Daniel-Searle**
## Display re-application date from test-results on IVA30 and MSVA30

Added re-application date to the IVA fail certificate data and into IVA and MSVA certificate so that I can be displayed.
[link to ticket number](https://dvsa.atlassian.net/browse/CB2-12602)

## Checklist

- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/main/java/uk/gov/dvsa/model/cvs/certificateData/IvaFailCertificateData.java [hunk 1]
```diff
@@ -145,4 +137,12 @@ public RequiredStandard[] getRequiredStandards() {
     public void setRequiredStandards(RequiredStandard[] requiredStandards) {
         this.requiredStandards = requiredStandards;
     }
+
```
Comment (cb-cs) [inline src/main/java/uk/gov/dvsa/model/cvs/certificateData/IvaFailCertificateData.java]: Did this class need to be modified as the reapplicationDate was already there?

Comment (Daniel-Searle) [reply src/main/java/uk/gov/dvsa/model/cvs/certificateData/IvaFailCertificateData.java]: I'll revert it now

**Comment by cb-cs** [comment]
@Daniel-Searle  - do we need to be bumping the version of template, as this will be in the next release?

File: src/test/java/pdfverification/tests/IVA30Tests.java [hunk 2]
```diff
@@ -37,6 +38,9 @@ public IVA30Tests() {
     public void setup() throws IOException {
         pdfData = pdfGenerationService.generate(htmlGenerator.generate(iva30));
         pdfReader = pdfParser.readPdf(pdfData);
+        FileOutputStream fileOutputStream  = new FileOutputStream("dan.pdf");
```
Comment (LGin-BJSS) [inline src/test/java/pdfverification/tests/IVA30Tests.java]: Debug code needs removing

**Review by naathanbrown — APPROVED** [review]

**Review by kcarr-bjss — APPROVED** [review]

**Pull request merged**